### PR TITLE
controllers testing

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments :
Screenshot from 2021-09-17 23-11-10

### CODE REVIEW

**Comment by maiducquang** [comment]
ready

**Comment by maiducquang** [comment]
ready

**Comment by ghost** [comment]
ready

File: resources/views/errors/minimal.blade.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>@yield('title')</title>
+
+        <!-- Fonts -->
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
+
+        <style>
```
Comment (ghost) [inline resources/views/errors/minimal.blade.php]: ssao lại có style trong blade template thế này

**Comment by ghost** [comment]
ready

**Review by vuminhhieu1311 — APPROVED** [review]

**Pull request merged**